src/containers/Confirm/Address/index.tsx

```diff
@@ -0,0 +1,303 @@
+import cr from 'classnames';
```

Comment:
Please, add tests for this container

Reply:
this is useless

Reply:
I disagree it's useless
You need to connect your components to Redux and simulate dispatching of actions and check the actions you call